src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -1816,11 +1832,11 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- MC-17014-->
 
-*  User can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
+*  You can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
 
 <!--- ENGCOM-5919-->
 
-*  Inconsistent sorting of FPT and tax totals has been resolved on the Admin order, create invoice, invoice, create credit memo,  and credit memo pages. *Fix submitted by Mahesh Singh in pull request [24663](https://github.com/magento/magento2/pull/24663)*. [GitHub-24225](https://github.com/magento/magento2/issues/24225)
+*  Inconsistent sorting of fixed product tax  (FPT and tax totals has been resolved on the Admin order, create invoice, invoice, create credit memo,  and credit memo pages. *Fix submitted by Mahesh Singh in pull request [24663](https://github.com/magento/magento2/pull/24663)*. [GitHub-24225](https://github.com/magento/magento2/issues/24225)
```

Comment:
A couple double spaces: `tax  (` and `memo,  and`.  One more above on line 1825: tax . (

Reply:
Thank you, but no time to groom for double spaces that do not affect meaning or display.